go-controller/pkg/ovn/egressip_test.go

```diff
@@ -5094,8 +5094,15 @@ var _ = ginkgo.Describe("OVN master EgressIP Operations", func() {
 					EgressIPs: []string{
 						updatedEgressIP.String(),
 					},
+					// PodSelector: metav1.LabelSelector{
+					// 	MatchLabels: egressPodLabel,
+					// },
+					// https://issues.redhat.com/browse/OCPBUGS-24055
```

Comment:
nit: I would refrain from adding JIRA bug links to upstream code (jira is very redhat specific) ; unless there is some TODO in place you want to get back to.
Also why are we commenting this code?

Reply:
+1
Additionally that jira card is not public

Reply:
> +1 Additionally that jira card is not public

TY both. I will take care of this soon.